website/.nvmrc

```diff
@@ -1 +1 @@
-v11.4.0
+v12.16.2
```

Comment:
Is there a reason we shouldn't use 13?

Reply:
I figured it would be better to target the latest LTS version to make sure we never run into problems with our dependencies. However, I'm not that versed into the Node world so maybe it's an unnecessary precaution.

Reply:
As discussed orally, it's safe to use 13.13.0 directly, I made the change 👍